site/content/docs/howto/configure-concierge-webhook.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Configure the Pinniped Concierge to validate webhook tokens
+title: Configure the Pinniped Concierge to Validate Webhook Tokens
```

Comment:
It's not hugely important to me but these were deliberately sentence case, following some common developer style guides such as [Google](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings) and [Microsoft](https://docs.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings)

Reply:
Oops. I was just trying to make them consistent. I just made all titles and all section headers in all how-to docs use sentence case in f4829178b35eaec559b9f8c29ac92ff27444ad5b.